io.github.sxyazi.yazi.appdata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>io.github.sxyazi.yazi</id>
+
+  <name>Yazi</name>
+  <developer id="io.github.sxyazi">
+    <name>三咲雅 · Misaki Masa</name>
+    <url>https://github.com/sxyazi</url>
+  </developer>
+  <summary>Terminal file manager</summary>
+  <summary xml:lang="zh_CN">终端文件管理器</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+
+  <!--Taken from README of official GitHub repo. MIT license. -->
+  <description>
+    <p>
+      Yazi (means "duck") is a terminal file manager written in Rust and based on non-blocking async I/O. It aims to provide an efficient, user-friendly, and customizable file management experience.
+    </p>
+  </description>
+
+  <recommends>
+    <control>keyboard</control>
+  </recommends>
+
+  <provides>
+    <binary>yazi</binary>
+    <binary>ya</binary>
+  </provides>
+
+  <screenshots>
+    <screenshot type="default">
+      <!--Does not yet exist but will be created upon acceptance to flathub-->
+      <image>https://raw.githubusercontent.com/flathub/io.github.sxyazi.yazi/master/screenshot.png</image>
```

Comment:
this should NEVER reference the tip of a branch.

Reply:
It has been changed to a screenshot on my fork of upstream. I'll change it to the upstream version once a PR gets merged.

Reply:
also the repo doesn't exist so the appstream validation will fail.

Reply:
I used a wrong link for the image, but the current link (on commit [dc03863](https://github.com/flathub/flathub/pull/5967/commits/dc038639a166220b6d18992d3c3cffe80b8fe13e)) should work.